{
  try {
    return input(ii).content();
  }
 catch (  final IOException ex) {
    throw error.get(ii,ex);
  }
}
